client/state/plugins/wporg/reducer.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * External dependencies
+ */
+import debugFactory from 'debug';
+import React from 'react/addons';
```

Comment:
Since this is a fresh PR, might as well switch to importing from `'react'`; see https://github.com/Automattic/wp-calypso/pull/2020/files — this follows the move to 0.14.